printer_mods/Koios/6mm_Glass/README.md

```diff
@@ -0,0 +1,10 @@
+# Glass mount
+
+This is a modification of the front panel locks from 2.1 to support holding 6mm tempered glass panels.  I used 3m VHB to adhere them to my glass panels so that thickness was considered as well.  The tab which supports the glass was thickened and has proven to be very robust on my printer. This can be run with 2.4 without issue, using panel sizes from the 2.1 bom.
+
+I've also added 'glass rest.stl' & the step file to secure the side panels with quick releases as well.  This is necessary as the glass is quite heavy and the lock levers wont hold it from sliding.
+
+Print the stock lock 2.1 lever's (here) to go along with these parts.
```

Comment:
Is `here` supposed to be a link to the 2.1 levers?  Also, remove apostrophe from `levers`